src/models/Class.js

```diff
@@ -1,29 +1,33 @@
-const mongoose = require('mongoose');
-
-const { Schema } = mongoose;
-
-const classSchema = new Schema({
-  hour: {
-    type: String,
-    required: true,
-  },
-  day: {
-    type: String,
-    required: true,
-  },
-  trainer: {
-    type: String,
-    required: true,
-  },
-  activity: {
-    type: String,
-    required: true,
-    enum: Schema.Types.ObjectId,
-  },
-  slots: {
-    type: Number,
-    required: true,
-  },
-});
-
-module.exports = mongoose.model('Class', classSchema);
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const classSchema = new Schema({
+  hour: {
+    type: String,
+    required: true,
+    match: /^([01]?[0-9]|2[0-3]):([0-5][0-9])$/,
+  },
+  day: {
+    type: String,
+    required: true,
+    match: /^(0?[1-9]|[12][0-9]|3[01])\/(0?[1-9]|1[0-2])\/\d{4}$/,

```

Comment:
Delete this line and update classSchema according to changes indicated on validations file